.github/workflows/install-and-build-pr.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   build-and-test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
> I am just wondering why this change is needed here because `ubuntu-latest` is working with the previous config. Would you clarify us?

I am just thinking ankr rpc seems not working due to [aBNB liquidity drain](https://cointelegraph.com/news/trader-allegedly-saw-over-5000x-gains-after-ankr-protocol-hack) they said their infra would not affect by that incident but actually I've been experiencing its not working quite frequently.

Reply:
> I am just wondering why this change is needed here because `ubuntu-latest` is working with the previous config. Would you clarify us?

Pretty sure somebody else fixed this within the last week due to some other CI/CD bug. Perhaps check git blame.

Reply:
> I am just wondering why this change is needed here because `ubuntu-latest` is working with the previous config. Would you clarify us?

I've already linked one related issue on foundry repo

Reply:
> I am just thinking ankr rpc seems not working due to [aBNB liquidity drain](https://cointelegraph.com/news/trader-allegedly-saw-over-5000x-gains-after-ankr-protocol-hack) they said their infra would not affect by that incident but actually I've been experiencing its not working quite frequently.

I have also experienced that ankr goerli rpc doesn't work when verifying diamond contract
so used this `https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161` on **https://rpc.info/**

Reply:
It looks like infura.io endpoint which could reach out the maximum limit asap or have CORS issue.